src/pages/HomePage/HomePage.js

```diff
@@ -290,6 +292,7 @@ HomePage.propTypes = {
 
 const mapStateToProps = state => ({
 	user: state.user.data.email,
+	username: state.user.data.name,
```

Comment:
It's better to use `get` from lodash to avoid `Can't read property x of undefined errors`.
You can have something like that `get(state, 'user.data.email')` instead of `state.user.data.email`, later can crash the app if `data` or `user` is undefined.

Reply:
Fixed 👍